tests/ert/unit_tests/forward_model_runner/test_fm_dispatch.py

```diff
@@ -484,3 +484,20 @@ async def wait_for_msg(msg_type):
             event_from_json(zmq_server.messages[-1]).error_msg
             == FORWARD_MODEL_TERMINATED_MSG
         )
+
+
+async def test_fm_dispatch_main_signals_sigterm_on_exception(capsys):
+    def mock_fm_dispatch_raises(*args):
+        raise RuntimeError("foo bar")
```

Comment:
Could you use something else than "foo bar"  ; eg. "forward model critical error" or something :)